FosServer/src/main/java/com/example/fostracker/servlets/VerificationServlet/VerificationDatabaseHelper.java

```diff
@@ -97,16 +98,29 @@ public static ResultSet queryData() {
      *
      * @return ResultSet object that refers to all rows in verifications table
      */
-    public static ResultSet queryStoreAndStatusData() {
+    public static ResultSet queryStoreAndStatusDataByEmail(String email) {
 
-        ResultSet storeAndStatusData =
-                SpannerClient.getDatabaseClient()
-                        .singleUse()
-                        .executeQuery(Statement.of("SELECT " + COLUMN_STORE_PHONE + ", "
-                                + COLUMN_VERIFICATION_STATUS
-                                + " FROM " + TABLE_NAME));
+        ResultSet storeAndStatusData;
+        Statement statement =
+                Statement.newBuilder(
+                        "SELECT " + COLUMN_STORE_PHONE +
+                                ", " + COLUMN_VERIFICATION_STATUS
+                                + " FROM " + TABLE_NAME
+                                + " WHERE " + COLUMN_AGENT_EMAIL
+                                + " = @" + QUERY_AGENT_EMAIL)
```

Comment:
Rewrite like this
"SELECT " + COLUMN_STORE_PHONE 
`+`   ", " + COLUMN_VERIFICATION_STATUS
`+`  " FROM " + TABLE_NAME
`+`  " WHERE " + COLUMN_AGENT_EMAIL 
`+`  " = @" + QUERY_AGENT_EMAIL)

Reply:
formatted